dist/rzslider.js

```diff
@@ -1268,6 +1269,20 @@
           this.scope.barStyle = {
             backgroundColor: color
           };
+        } else if (this.options.selectionBarGradient) {
+          var offset = this.options.showSelectionBarFromValue !== null ? this.valueToPosition(this.options.showSelectionBarFromValue) : 0,
+            reversed = offset - position > 0 ^ isSelectionBarFromRight,
+            direction = this.options.vertical ? 'top' : (reversed ? 'left' : 'right');
```

Comment:
the vertical + reversed case is not handled I think. Look this demo: http://jsfiddle.net/nLk7p00v/